docs/security/trust/trust_sandbox.md

```diff
@@ -212,60 +150,57 @@ Now, you'll pull some images.
 
 2. Tag it to be pushed to our sandbox registry:
 
-        # docker tag docker/trusttest sandboxregistry:5000/test/trusttest:latest
+        / # docker tag docker/trusttest sandboxregistry:5000/test/trusttest:latest
 
 3. Enable content trust.
 
-        # export DOCKER_CONTENT_TRUST=1
+        / # export DOCKER_CONTENT_TRUST=1
 
 4. Identify the trust server.
 
-        # export DOCKER_CONTENT_TRUST_SERVER=https://notaryserver:4443
+        / # export DOCKER_CONTENT_TRUST_SERVER=https://notaryserver:4443
 
     This step is only necessary because the sandbox is using its own server.
     Normally, if you are using the Docker Public Hub this step isn't necessary.
 
 5. Pull the test image.
 
-        # docker pull sandboxregistry:5000/test/trusttest
+        / # docker pull sandboxregistry:5000/test/trusttest
         Using default tag: latest
-        no trust data available
+        Error: remote trust data does not exist for sandboxregistry:5000/test/trusttest: notaryserver:4443 does not have trust data for sandboxregistry:5000/test/trusttest
 
-      You see an error, because this content doesn't exist on the `sandboxregistry` yet.
+      You see an error, because this content doesn't exist on the `notaryserver` yet.
 
 6. Push the trusted image.
 
-        # docker push sandboxregistry:5000/test/trusttest:latest
-        The push refers to a repository [sandboxregistry:5000/test/trusttest] (len: 1)
-        a9539b34a6ab: Image successfully pushed
-        b3dbab3810fc: Image successfully pushed
-        latest: digest: sha256:1d871dcb16805f0604f10d31260e79c22070b35abc71a3d1e7ee54f1042c8c7c size: 3348
+        / # docker push sandboxregistry:5000/test/trusttest:latest
```

Comment:
Perhaps for this step, we should mention that they need to create passwords? (I realize the output mentions it, but it may be good to tell them upfront)